.travis/runtox

```diff
@@ -16,6 +16,25 @@ ME=$(basename $0)
 SCRIPTDIR=$(readlink -f $(dirname $0))
 BANNERSIZE=90
 
+# LSR_MOLECULE_DOCKER_VERSION is the version string to pass
+# to pip install - this is usually a comparison operator
+# followed by a version number e.g. '<4.3' but in reality
+# may be anything that pip allows to follow a package name
+# in an install specification
+if [ -z "${LSR_MOLECULE_DOCKER_VERSION:-}" ]; then
+  if type docker > /dev/null 2>&1; then
+    DOCKER_SERVER_API_VERSION=$(docker version --format '{{.Server.APIVersion}}')
+    case "$DOCKER_SERVER_API_VERSION" in
+    1.39|1.[4-9]*) LSR_MOLECULE_DOCKER_VERSION="";;
+    1.3[0-8]|1.[0-2]*) LSR_MOLECULE_DOCKER_VERSION='<4.3';;
+    *) echo Error: unknown docker server api version "$DOCKER_SERVER_API_VERSION"; exit 1;;
```

Comment:
The if works if we only need to care about that version. Otherwise the case pattern could still be used:
```suggestion
    1.3[0-8]|1.[0-2]*) LSR_MOLECULE_DOCKER_VERSION='<4.3';;
    *) LSR_MOLECULE_DOCKER_VERSION="";;
```

Reply:
done in https://github.com/linux-system-roles/template/pull/40/commits/3797f7c1a5be132c4da887d377e639b924c216ae